packages/gatsby/cache-dir/loader.js

```diff
@@ -1,36 +1,40 @@
-import React, { createElement } from "react"
 import pageFinderFactory from "./find-page"
 import emitter from "./emitter"
```

Comment:
this module has the most changes and most likely to have broken something. I removed unused code paths for the time being to simplify it and save some bytes. I also trimmed back the public api so it's easier to see what is being used and hopefully make changes in the future. I don't think I changed anything important in terms of functionality. It fetches page resources, and lets you prefetch them by path

Reply:
Looked good from my review on my phone! Love how much simpler promises make it.